API/Routes/ViewMemoryRouter.js

```diff
@@ -0,0 +1,8 @@
+import ViewMemoryController from "../Controllers/ViewMemoryController";
+import express from 'express';
+
+const ViewMemoryRouter = express.Router()
+
+ViewMemoryRouter.get('/api/v1/memory', ViewMemoryController.FetchViewMemory);
```

Comment:
```suggestion
ViewMemoryRouter.get('/api/v1/memory', ViewMemoryController.fetchMemory);
```
Can you change this to camelCase. `fetchMemory`
I also think `fetchMemory` is a better naming for this than `fetchViewMemory` since there is no visible UI for a user to see.